src/radical/pilot/constants.py

```diff
@@ -60,7 +61,7 @@
 #
 # Flags
 #
-CREATE_PARENTS = 'CreateParents'  # Create parent directories while writing file
+CREATE_PARENTS = rsf.CREATE_PARENTS  # Create parent directories while writing file
 SKIP_FAILED    = 'SkipFailed'     # Don't stage out files if tasks failed
```

Comment:
either all or none of the flags should be strings.

Reply:
Agreed. I would suggest none to be string and we move to using integers as we do in saga.